lighthouse-core/config/seo.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  extends: 'lighthouse:default',
+  groups: {
+    'seo-mobile': {
+      title: 'Mobile Friendly',
+      description: 'Having a mobile-friendly website is a critical part of your online presence. In many countries, smartphone traffic now exceeds desktop traffic. '
+    },
+    'seo-content': {
+      title: 'Content Best Practices',
+      description: 'Ensure your markup enables crawlers to better understand your app\'s content.'
```

Comment:
the distinction between this and "Crawling and Indexing" is a little blurry to me, is this basically that you use proper tags, have title, etc while crawling is more like pushState, not using `#!/` paths, etc?